packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -48,6 +46,7 @@ export class AtomicSearchBox implements AtomicSearchBoxOptions {
   );
 
   private engine!: Engine;
+  private i18n!: i18n;
```

Comment:
It would be easier to scale Atomic if we grouped all these global instances under a single object. Every time we have to add another one, we will just need to update the initialization decorator and the object's interface. We will not need to add a new property in every component, or import the interface for that property.

```
interface Context {
    engine: Engine;
    logger: Logger;
    i18n: i18n;
}

// atomic-search-box.tsx

private context: Context;

render() {
    ...
    this.context.i18n.t(...);
}


```

Reply:
Yep, will do this in a separate PR, as it will affect all components